### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by BenMukebo**
## In this Milestone, I  add forms to my Blog app.

- Create forms to perform the following functions:
  - Creates a Post on behalf of the current_user.
  - Create a comment on behalf of the current_user.
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  def create
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:post_id])
+    @like = @post.likes.new
+    @like = @user.likes.new(post_id: @post.id, author_id: @user.id)
+
+    if @like.save
+      redirect_to user_post_path(@user, @post)
+    else # rubocop:todo Style/EmptyElse
+      # render 'new'
```
Comment (arslanbisharat) [inline app/controllers/likes_controller.rb]: - [ ] Let's remove the commented code and rubocop command

File: db/migrate/20220125101325_create_likes.rb [hunk 2]
```diff
@@ -5,6 +5,8 @@ def change
       t.references :post, null: false, foreign_key: true
 
       t.timestamps
+
+      # add_index :likes, [:author_id, :post_id], unique: true
```
Comment (arslanbisharat) [inline db/migrate/20220125101325_create_likes.rb]: - [ ] Let's remove commented code

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: app/controllers/likes_controller.rb (hunk 1 above)
Comment (BenMukebo) [reply app/controllers/likes_controller.rb]: Thanks to much, I've removed it!!

File: db/migrate/20220125101325_create_likes.rb (hunk 2 above)
Comment (BenMukebo) [reply db/migrate/20220125101325_create_likes.rb]: Thanks to much, I've removed it!!

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @BenMukebo🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**